oshmem/include/pshmem.h

```diff
@@ -14,32 +14,7 @@
 #ifndef PSHMEM_SHMEM_H
 #define PSHMEM_SHMEM_H
 
-
-#include <stddef.h>     /* include for ptrdiff_t */
-#include <stdint.h>     /* include for fixed width types */
-#if defined(c_plusplus) || defined(__cplusplus)
-#    include <complex>
-#    define OSHMEM_COMPLEX_TYPE(type)    std::complex<type>
-#else
-#    include <complex.h>
-#    define OSHMEM_COMPLEX_TYPE(type)    type complex
-#endif
-
-
-#ifndef OSHMEM_DECLSPEC
-#  if defined(OPAL_C_HAVE_VISIBILITY) && (OPAL_C_HAVE_VISIBILITY == 1)
-#     define OSHMEM_DECLSPEC __attribute__((visibility("default")))
-#  else
-#     define OSHMEM_DECLSPEC
-#  endif
-#endif
-
-#if defined(__STDC_VERSION__) && (__STDC_VERSION__ >= 201112L)
-#define OSHMEMP_HAVE_C11 1
```

Comment:
if we remove this block, then we have to use the `OSHMEM_HAVE_C11` (note there is no `P`) macro (that is defined in `shmem.h`). @yosefe is that OK for you ?

Reply:
nice catch :+1:

Reply:
@yosefe do you take care of it or should I file a new pr?

Reply:
@bertwesarg I just filed #6183